boards/t430-hotp-maximized/t430-hotp-maximized.config

```diff
@@ -0,0 +1,86 @@
+# Configuration for a T430 running Qubes and other Linux Based OSes (through kexec)
+#
+# Includes 
+# - Deactivated+neutered ME and expended consequent IFD BIOS regions 
```

Comment:
NIT: "expended". I assume you mean expanded or extended.  This typo is in many files, so please get them all.